keyvi/src/cpp/dictionary/fsa/automata.h

```diff
@@ -47,12 +48,14 @@ namespace keyvi {
 namespace dictionary {
 namespace fsa {
 
-
 class Automata
 final {
 
    public:
-    Automata(const char * filename, bool load_lazy=false) {
+    Automata(const char * filename, bool load_lazy):
+      Automata(filename, load_lazy ? default_loading_strategy : loading_strategy_types::populate) {}
```

Comment:
LGTM. I am only concerned if u notice a slowdown in dictionary loading because of this change. MAP_POPULATE is slow from the last time I tested it.